test/common/testlib.py

```diff
@@ -94,7 +99,7 @@ def attach(filename):
 
 
 class Browser:
-    def __init__(self, address, label, port=None):
+    def __init__(self, address, label, pixels_label, port=None):
```

Comment:
It would be safe to let that default to None, AFAICS. That would disable pixel testing, right? The code below already relies on that anyway.

Reply:
Yes, good point.